classes/db/DbPDO.php

```diff
@@ -99,12 +99,12 @@ public function connect()
         try {
             $this->link = $this->_getPDO($this->server, $this->user, $this->password, $this->database, 5);
         } catch (PDOException $e) {
-            die(sprintf(Tools::displayError('Link to database cannot be established: %s'), utf8_encode($e->getMessage())));
+            throw new PrestaShopException(sprintf(Tools::displayError('Link to database cannot be established: %s'), utf8_encode($e->getMessage())));
```

Comment:
That's too bad a PrestaShopException cannot display details of the previous Exception (if given), we could have written

```php
throw new PrestaShopException('Link to database cannot be established.', 0, $e);
```